configure.ac

```diff
@@ -6,7 +6,7 @@
 #
 
 # Initialization
-AC_INIT([aws-ofi-nccl], [GitHub-dev], [al-ofi-nccl-team@amazon.com], , [http://github.com/aws/aws-ofi-nccl])
+AC_INIT([aws-ofi-nccl], [1.9.0a1], [al-ofi-nccl-team@amazon.com], , [http://github.com/aws/aws-ofi-nccl])
```

Comment:
Can we make this semver compatible?

Reply:
We've historically used the more GNU-ish a1 format than the preferred semantic versioning grammar and I'm not really sure there's a good reason to change right now.  All the tools that do version number parsing are ok with the GNU-style, because GNU.